src/js/modules/articles/components/ArticleRow.js

```diff
@@ -63,11 +68,15 @@ const styles = {
   }
 };
 
-/*
-BUG THAT CAUSES LINK TO ARTICLE TO NOT INCLUDE SUBSECTION HAS BEEN FIXED IN FEATURE/MEDIA-MODULE
- */
-
-const ArticleRow = ({ classes, article, featuredMedia, section }) => {
+const ArticleRow = ({ classes, article, sections, media }) => {
+  const matchedSections = Object.filter(sections, sectionObject => {
```

Comment:
Wait what. Why are you filtering if you're searching by section slug? Aren't the object keys the section slugs?